applications/IgaApplication/custom_modelers/iga_modeler.cpp

```diff
@@ -156,8 +158,17 @@ namespace Kratos
                 }
             }
 
-            rGeometryList[i].CreateQuadraturePointGeometries(
-                geometries, shape_function_derivatives_order, integration_info);
+            if (rGeometryType == "SurfaceEdge"
```

Comment:
couldn't you check this with the KRatosGeometryType?

Reply:
Unfortunately, this doesn't work as those are actually different types. Those come from B-Rep terms and may not necessarily have a geometry counterpart in Kratos.